pkg/kubernetes/config/kubeconfig.go

```diff
@@ -2,12 +2,15 @@ package config
 
 import (
 	"fmt"
+	"log"
```

Comment:
should this be:
```suggestion
	log "github.com/sirupsen/logrus"
```
?

Just making sure your IDE didn't flub an auto import.

Reply:
Absolutely! Good catch. Thank you!